token-metadata/program/tests/utils/programmable.rs

```diff
@@ -136,6 +163,34 @@ pub async fn create_default_metaplex_rule_set(
         .add(sale_delegate_operation.to_string(), transfer_rule.clone())
         .unwrap();
 
+    if use_delegate_allow_list {
```

Comment:
was wondering why this boolean is here

Reply:
The boolean is used for a test to choose to whether add a program allow list for the delegate scenarios or not.